src/gui/systray.h

```diff
@@ -103,15 +102,23 @@ class Systray
     void openHelp();
     void shutdown();
 
+    // These window signals are listened to in Window.qml
```

Comment:
I disagree on the usefulness of this comment
you make an assumption about the use that might not be necessary true

Reply:
and I agree that the current API is confusing
think of `showWindow` and `openMainDialog`
which one does what ?
which one to connect to ?

Reply:
To clarify, it is not clear in any way how emitting these signals leads to the tray window being opened. Hence the comment providing at least some sort of pointer

Reply:
I definitely understand your point
I am concerned with the long term maintenance of those comments ?
what if we would make our APIs easier to understand by reengineering that part ?

Reply:
but should we not at least try a little bit harder
maybe `showWindow` could become something like `requestWindowShown`
`hideWindow` could become something like `requestWindowHidden`

Reply:
I think the smartest solution here would be to re-engineer this, IMO showing/hiding the window should all be handled directly by the sys tray and not by the Window qml

Do you think this would be a good idea to change in a new PR?

Reply:
let's do another PR with cleanup/improvements to the API